flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -883,25 +713,33 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this.allSelected = false;
         Coherent.trigger('UNFOCUS_INPUT_FIELD');
         this._inOutElement.style = null;
-        this.getChildById("header").style = null;
+        this.root.querySelector("#header").style = null;
         if (this.check_focus) {
             clearInterval(this.check_focus);
         }
     }
 
+    // TODO handle side properly
```

Comment:
⚠

Reply:
That is actually done, just the comment wasn't removed :D